### ISSUE TITLE
Addresses truncated to 32 bit on 64bit system

### ISSUE THREAD

**Issue by PtrMan** [KhronosGroup/Vulkan-LoaderAndValidationLayers#831]
vulkan debug: layerprefix=DS message=Queue 0x102ba7cc8 is signaling semaphore 0x11 that has already been signaled but not waited on by queue 0x495f455404ea5378.
vulkan debug: layerprefix=ParameterValidation message=vkQueueSubmit: returned VK_ERROR_VALIDATION_FAILED_EXT, indicating that API validation has detected an invalid use of the API
t{0} vkQueueSubmit(queue = 0x2ba7cc8, submitCount = 0x1,

the last line from the validation layer prints it in the wrong way

**Comment by mark-lunarg** [comment]
The last message with the truncated object handle is coming from APIdump, and needs to be addressed there.

**Comment by karl-lunarg** [comment]
I cannot reproduce this.  Something may have changed since the bug was filed.

If I apply the following hack, to make sure a handle value is larger than 32 bits:

```
diff --git a/vk-vtlayer-generate.py b/vk-vtlayer-generate.py
index 5e3873a..a39a6ad 100755
--- a/vk-vtlayer-generate.py
+++ b/vk-vtlayer-generate.py
@@ -1045,6 +1045,11 @@ class ApiDumpSubcommand(Subcommand):
             stmt = "    return result;\n"
         f_open = 'loader_platform_thread_lock_mutex(&printLock);\n    '
         log_func = '%s\n' % self.lineinfo.get()
+        # Start Change
+        p = proto.params[0]
+        if p.name == "queue":
+            log_func += 'queue=(VkQueue)0x22201234567;'
+        #End Change
         log_func += '    if (StreamControl::writeAddress == true) {'
         log_func += '\n        (*outputStream) << "t{" << getTIDIndex() << "} f{" << g_frameCounter << "} vk%s(' % proto.name
         log_func_no_addr = '\n        (*outputStream) << "t{" << getTIDIndex() << "} f{" << g_frameCounter << "} vk%s(' % proto.name
```

the api_dump layer emits the expected non-truncated value for queue:

```
t{0} f{0} vkQueueSubmit(queue = 0x22201234567, submitCount = 0x1, pSubmits = 0x7ffc06c78370, fence = 0x1645de0) = VK_SUCCESS
   pSubmits[0]:
    sType = VK_STRUCTURE_TYPE_SUBMIT_INFO
    pNext = 0x0
    waitSemaphoreCount = 0x0
    pWaitSemaphores = 0x0
    pWaitDstStageMask = 0x0
    commandBufferCount = 0x1
    pCommandBuffers = 0x7ffc06c783c0
    signalSemaphoreCount = 0x0
    pSignalSemaphores = 0x0
     pCommandBuffers[0] = 0x162ea40

```

Please reopen if this doesn't seemed fixed in SDK 1.0.24 or the latest code from the VulkanTools repo.
